src/keymap.cpp

```diff
@@ -484,6 +484,14 @@ static const std::vector<OpDesc> opdescs = {
 	{OP_INT_SET, "set", "internal-set", "", KM_INTERNAL},
 
 	{OP_INT_GOTO_URL, "gotourl", "internal-goto-url", "", KM_INTERNAL},
+
+	{
+		OP_PREVSEARCHRESULTS,
+		"prevsearchresults",
+		"z",
+		translatable("Previous search results"),
```

Comment:
This is an action, so let's describe it as one: "Return to previous search results (if any)".